### ISSUE TITLE
Add quitter

### ISSUE THREAD

**Issue by daniele-athome** [kontalk/androidclient#966]
https://quitter.se/kontalk

**Comment by tetris4** [comment]
Please also add identica while at it:
https://identi.ca/kontalknet

**Comment by daniele-athome** [comment]
> Please also add identica while at it:
https://identi.ca/kontalknet

Isn't that already present?

**Comment by tetris4** [comment]
I only see at the bottom right: facebook, twitter, google plus, github.

**Comment by daniele-athome** [comment]
I've updated the relevant issue at [website](https://github.com/kontalk/website/issues/29).

**Comment by daniele-athome** [comment]
I also believe quitter.se has closed. quitter.no will also close in September.
But there are other instances: https://gnu.io/social/try/

I believe just creating an account on any of those would do. @tetris4 am I right?

**Comment by tetris4** [comment]
Yes it has been unfortunately down for a while now. At this moment, I think going for a mastodon account would be the best option.
